src/FSharp.Formatting.CommandTool/BuildCommand.fs

```diff
@@ -643,33 +683,40 @@ type CoreBuildOptions(watch) =
                 if not docsQueued then
                     docsQueued <- true
                     printfn "Detected change in '%s', scheduling rebuild of docs..."  this.input
-                    Async.Start(async {
+                    async {
                         do! Async.Sleep(300)
                         lock monitor (fun () ->
                         docsQueued <- false
                         if runDocContentPhase1() then
                             if runDocContentPhase2() then
                                 regenerateSearchIndex()
-                        ) }) )
+                        )
+                    }
+                    |> Async.RunSynchronously
```

Comment:
This has changed from Start to RunSynchronously.  If it's RunSynchronously then no async is needed?

Reply:
You are right, this is unnecessary. I was kind of thrown off by the 'fire and forget' nature of Async.Start (via [this](https://docs.microsoft.com/de-de/dotnet/fsharp/tutorials/asynchronous-and-concurrent-programming/async#asyncstart)) and ditched it without second thought thinking i have to wait until the computation is finished. I missed that the hot reload trigger does not have to happen after the whole block but just after the last function call inside it, so i moved it inside the block now using the original Async.start version again.